Workbooks/WorkbooksMetadata.json

```diff
@@ -1270,5 +1270,18 @@
     "templateRelativePath": "WorkspaceUsage.json",
     "subtitle": "",
     "provider": "Azure Sentinel community"
+  },
+  {
+    "workbookKey": "McAfeeePOWorkbook",
+    "logoFileName": "mcafee_logo.svg",
```

Comment:
Please add the logo under workbooks/images/logos. You can refer the instructions here https://github.com/Azure/Azure-Sentinel/tree/master/Workbooks (step 2)
(optional) A logo that you want the workbook to display. Place it under workbooks/images/logos - if not supplied - it will be the generic workbooks logo.
This logo should be in SVG format.